lib/api.js

```diff
@@ -102,3 +102,42 @@ builder.declare({
   res.reply();
 });
 
+builder.declare({
+  method: 'post',
+  route: '/worker-configurations/:id',
```

Comment:
Same thing with `:id` here.

For reference our current list of scope params is:

```
     22 <workerType>
     15 <provisionerId>
     11 <workerId>
     11 <workerGroup>
     11 <schedulerId>
     10 <taskId>
      9 <taskGroupId>
      8 <name>
      8 <hookId>
      8 <hookGroupId>
      8 <account>
      6 <scope>
      6 <runId>
      6 <clientId>
      3 <table>
      3 <roleId>
      3 <property>
      3 <prefix>
      3 <container>
      3 <bucket>
      2 <route>
      2 <repo>
      2 <project>
      2 <priority>
      2 <owner>
      2 <namespace>
      1 <user>
      1 <routingKey>
      1 <channel>
      1 <cacheName>
      1 <address>
```

Obviously some of those will need to be updated as well if we try to make them useful.